shoes-core/lib/shoes/arc.rb

```diff
@@ -1,9 +1,24 @@
 # frozen_string_literal: true
 
 class Shoes
+  # Arc.  A basic element representing a curve of a circle or an oval.
+  #
+  # @example A simple arc which describes the bottom half of a circle and uses the centered style.
+  #   arc 200, 200, 100, 100, 0, Shoes::PI, center: true
+  # @example An arc which describes the top half of a circle.
+  #   arc 200, 200, 100, 100, Shoes::PI, 0
+  # @note Angle is the gradient angle used across all art elements. Angle1/2 are the angles of
+  #   the arc itself!
+  # @author Jason Clark
```

Comment:
Don't know that I deserve specific credit here--I probably just shuffled the class around from it's original location :)